Rakefile

```diff
@@ -8,6 +8,19 @@ Rails.application.load_tasks
 begin
   require 'cucumber/rake/task'
   Cucumber::Rake::Task.new(:default)
+
+  namespace :cucumber do
+    desc 'Analyse Javascript with JSHint and Javascript Code Style checker'
+    task :javascript_disabled do
+      puts '*' * 50
+      puts 'Running cucumber features with JS disabled...'
+      puts '*' * 50
+      system('WITHOUT_JS=true cucumber --tags @javascript')
```

Comment:
This made me scratch my head for a moment -- I think it's the conflicting environment variable and tag. Maybe this could be simplified by explicitly specifying a further tag on those feature files you want to run in both modes? Perhaps something like `@javascript @noscript`?

You could then simplify the runner command with `cucumber --tags @javascript,@noscript`